src/main/kotlin/com/terraformation/backend/customer/AppNotificationService.kt

```diff
@@ -261,6 +265,71 @@ class AppNotificationService(
         setOf(Role.Owner, Role.Admin))
   }
 
+  @EventListener
+  fun on(event: ParticipantProjectSpeciesApprovedSpeciesEditedEvent) {
+    systemUser.run {
```

Comment:
This is harmless but I don't think it's needed; the event will be published inside a `systemUser.run` block.

Reply:
I ended up having to add a permission allowance for reading global roles in the service test.